### PR TITLE
search

### PR DESCRIPTION
**Pull request by Tran-Khac-Nam**
## Related Tickets:
- [#30182 ID](https://edu-redmine.sun-asterisk.vn/issues/30182)

## WHAT this PR does?

search product by name

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.
simplescreenrecorder-2020-10-26_11 55 00(1)

### CODE REVIEW

**Comment by Tran-Khac-Nam** [comment]
ready

File: resources/js/customer/pages/search.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        console.log('click');
+        $.ajax({
+            type: "POST",
+            url: $(this).attr('data-url'),
+            data:'name=' + $(this).val(),
+            beforeSend: function() {
+            },
+            success: function(data) {
+                $(".search-data").html(data);
+                console.log(data);
```
Comment (huyenttm-0901) [inline resources/js/customer/pages/search.js]: bỏ console

File: resources/js/customer/pages/search.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+$(document).ready(function() {
+    $("#search-box").keyup(function() {
+        console.log('click');
```
Comment (huyenttm-0901) [inline resources/js/customer/pages/search.js]: tương tự

File: resources/js/customer/pages/search.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        $.ajax({
+            type: "POST",
+            url: $(this).attr('data-url'),
+            data:'name=' + $(this).val(),
+            beforeSend: function() {
+            },
```
Comment (huyenttm-0901) [inline resources/js/customer/pages/search.js]: ko dùng thì bỏ đi em

File: resources/lang/en/customer.php [hunk 4]
```diff
@@ -76,4 +76,13 @@
     'provisional' => 'Provisional',
     'into_money' => 'Into Money',
     'pay_now' => 'Pay Now',
+    'no_item_cart' => 'Your shopping cart is empty.',
+    'no_review' => 'No reviews from users yet. Let us know your opinion.',
+    'readmore' => 'Read more',
+    'no_result' => 'No results were found',
+    'Search' => 'Search',
+    'ops' => 'Oooops!',
+    'product_not_found' => 'Product Item Not Found',
```
Comment (huyenttm-0901) [inline resources/lang/en/customer.php:85]: viết file vi

**Review by huyenttm-0901 — COMMENTED** [review]
thêm attachment

**Comment by Tran-Khac-Nam** [comment]
ready

**Pull request merged**